frontends/web/src/routes/account/send/feetargets.tsx

```diff
@@ -30,6 +30,7 @@ interface FeeTargetsProps {
     disabled: boolean;
     fiatUnit: Fiat;
     proposedFee?: AmountWithConversions;
+    showPlaceholder?: boolean;
```

Comment:
better name would be good, not clear what placeholder means. maybe `showCalculatingFeeLabel`, or `processingTxProposal`?

Reply:
In context of the feetargets component it should pretty clear, but `processingTxProposal` or `showCalculatingFeeLabel ` is better 👍  

Btw. I just used an already existing text, "calculating fee estimation", maybe it should be another i18n entry and just say "calculating fee…" ?

Reply:
https://github.com/digitalbitbox/bitbox-wallet-app/compare/af5abb807f12fedbcecf1f161fe1d0aa508ed686..fff2e4afd83da4c8bb7605cbd17b20b1e147d3d9